Packs/DeveloperTools/Integrations/CustomIndicatorDemo/IndicatorFields/param1.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
should be under `Packs/DeveloperTools/IndicatorFields/param1.json`

Reply:
Fixed